qemu/tests/cfg/max_lun.cfg

```diff
@@ -0,0 +1,24 @@
+- max_lun:
+    virt_test_type = qemu
+    type = max_channel_lun
+    restart_vm = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
+    start_vm = no
+    only ppc64le ppc64
+    only Linux
+    spapr_vscsi_reg = 0x71000001
+    stg_image_name = "images/%s"
+    force_create_image_image1 = no
+    force_create_image = yes
+    channel = "channel=0"
+    bootindex_image1 = 0
+    error_info = "bad scsi device lun: 32"
+    variants luns:
+        - lun_32:
+            stg_image_num = 32
+        - lun_33:
+            stg_image_num = 32
```

Comment:
why not set stg_image_num =33?

Reply:
It will raise error,so use a type of workaround.